charts/cbcontainers-operator/README.md

```diff
@@ -0,0 +1,57 @@
+# cbcontainers-operator
+
+This is the official Helm chart for installation of the CBContainers Operator.
+
+## Installation
+
+The chart can be installed as is, without any customization or modifications.
+
+```sh
+helm repo add vmware TODO-chart-repo/TODO-chart-name
+helm repo update
+helm install cbcontainers-operator TODO-chart-repo/TODO-chart-name
+```
+
+or from source
+
+```sh
+cd charts/cbcontainers-operator
+helm install cbcontainers-operator ./cbcontainers-operator-chart
+```
+
+## Customization
+
+| Parameter                                              | Description                                                      | Default                                                                            |
+| ------------------------------------------------------ | ---------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| `spec.operator.image.repository`                       | The repository of the operator image                             | `cbartifactory/octarine-operator`                                                  |
+| `spec.operator.image.version`                          | The version of the operator image                                | `5.1.0`                                                                            |
+| `spec.rbacProxy.image.repository`                      | The repository of the Kube RBAC proxy image                      | `gcr.io/kubebuilder/kube-rbac-proxy`                                               |
+| `spec.rbacProxy.image.version`                         | The version of the Kube RBAC proxy image                         | `v0.8.0`                                                                           |
```

Comment:
I am not sure this is actually easily configurable since it is controlled by the version of the operator-framework. I.E. changing this could lead to problems with the operator.

I'd remove all config for the rbacProxy from parameters and leave that control to us (at least in theory; they can always try to change the definition in their cluster of course)